core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/TrafT4Statement.java

```diff
@@ -1247,7 +1247,7 @@ void setMultipleResultSets(int num_result_sets, TrafT4Desc[][] output_descriptor
 			if (desc == null) {
 				resultSet_[i] = null;
 			} else {
-				resultSet_[i] = new TrafT4ResultSet(this, desc, stmt_labels[i], true);
+				resultSet_[i] = new TrafT4ResultSet(this, desc, stmt_labels[i], this.sqlStmtType_==TRANSPORT.TYPE_CALL);
```

Comment:
this is the parameter pass to spj_rs which is always true  that cause setCursorName in TraT4Statement.java does not work,